contracts/incentives/StakingRewards.sol

```diff
@@ -51,6 +51,8 @@ contract StakingRewards is IRewardDistributionRecipient, ReentrancyGuard {
       * @param _stakingToken Staking token contract address
       * @param _initreward   Initial reward
       * @param _starttime    Start time
+      * @param _duration     Duration of the staking in days
```

Comment:
@kinspotting That does not seem accurate to me.  Otherwise why do we need `isActive`?